src/docs/release/breaking-changes/bottom-navigation-title-to-label.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Bottom Navigation Title To Label
+description: Deprecated BottomNavigationBarItem’s title (a Widget) in favor of label (a String). 
+---
+
+## Summary
+
+{% comment %}
```

Comment:
Not sure if these are meant to be removed or if they'll be invisible

Reply:
They'll be inviisible.

Reply:
While the comments are invisible, can you remove them from the file? They just add clutter.

Reply:
Done